frontend/playwright/ui/specs/tokens.spec.js

```diff
@@ -95,6 +95,24 @@ const setupTypographyTokensFile = async (page, options = {}) => {
   });
 };
 
+const checkInputFieldWithError = async (tokenThemeUpdateCreateModal, inputLocator) => {
+    await expect(inputLocator).toHaveAttribute("aria-invalid", "true");
+
+    const errorMessageId = await inputLocator.getAttribute("aria-describedby");
+    await expect(
+      tokenThemeUpdateCreateModal.locator("#" + errorMessageId),
```

Comment:
It is a matter of opinion, but IMHO template literals provide a better tool for text operations

```js
tokenThemeUpdateCreateModal.locator(`#${errorMessageId}`)
```